src/server/clients_info.rs

```diff
@@ -25,9 +21,13 @@ pub(crate) struct ClientBuffers {
     entities: Vec<Vec<Entity>>,
 }
 
+/// Stores meta-information about connected clients.
```

Comment:
I agree with the move, but could you revert it for now to avoid conflicts with my other branch? I will do it myself later :smile: